Assets/Mirror/Runtime/NetworkStartPosition.cs

```diff
@@ -11,14 +11,17 @@ namespace Mirror
     [HelpURL("https://mirror-networking.com/docs/Components/NetworkStartPosition.html")]
     public class NetworkStartPosition : MonoBehaviour
     {
+        public NetworkManager manager;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Make this field 'private' and encapsulate it in a 'public' property.](https://app.codacy.com/manual/paulpach/MirrorNG/pullRequest?prid=4841467)